src/Controller/SearchEngineController.php

```diff
@@ -62,12 +62,13 @@ public function searchPage (
 		$searchEngineManager->setTNTSearchConfiguration();
 
 		//Launch Search
-		$results = $searchEngineManager->search(implode($formObj['formTexts']['keywords'], ' '), $formObj['formFilters']['result_type'], $formObj['formFilters']['groups'], $formObj['formFilters']['particularGroups']);
+		$searchResults = $searchEngineManager->search(implode($formObj['formTexts']['keywords'], ' '), $formObj['formFilters']['result_type'], $formObj['formFilters']['groups'], $formObj['formFilters']['particularGroups']);
 
 		return $this->render( 'pages/search/search.html.twig', [
 			'form'    => $formObj['form']->createView(),
-			'result_number' => array_sum(array_map("count", $results)),
-			'results' => $results,
+			'result_number' => array_sum(array_map("count", $searchResults['results'])),
+			'results' => $searchResults['results'],
+			'isCurrentConnexion' => $searchResults['connexionBoolean'],
```

Comment:
C'est très bien comme ça, mais tu pouvais aussi utiliser le même nom partout :
```suggestion
			'isUserConnected' => $searchResults['isUserConnected'],
```
Voire ne pas faire transiter cette info :
```suggestion
			'isUserConnected' => (bool) $this->getUser(),
```